frontend/app/templates/components/mapbox/current-mouseover-features.hbs

```diff
@@ -0,0 +1,22 @@
+<Mapbox::CurrentMousePosition
```

Comment:
So, this component is like `FeatureSelector`, but instead of binding a click event it binds the mousePosition event, right? so should this be called like `FeatureHoverer`?

Reply:
Ahh, good point. That'll make it more consistent. Will change.